integration_tests/test_builtin_pow.py

```diff
@@ -22,6 +21,31 @@ def test_pow():
     b = -4
     print(pow(a, b))
 
+    i1: i64
+    i2: i64
+    i1 = 2
+    i2 = 5
+    assert pow(i1, i2) == 32
+    i1 = 6
+    i2 = -3
+    print(pow(i1, i2))
```

Comment:
Can we avoid printing and just use `assert`, because I think it does not make any sense to print them unless they aren't working properly(unnecessarily logs output which is hard to find if it is actually correct or not)?

Reply:
The issue is that when both the arguments are integers, the result should be an integer, except when the second argument is negative, the output is a floating-point number. 
Once we implement generics, we can deal with this.